dask/tests/test_threaded.py

```diff
@@ -7,6 +7,8 @@
 
 import pytest
 
+from distributed.utils_test import cleanup  # noqa: F401
```

Comment:
distributed is actually only an optional dependency of dask. This is why the mindeps build fails

Reply:
I think a separate PR that checks for leaked threads at the end of every test would be good here?

Reply:
Yeah, I think checking for leaks might generally be a good idea.